src/components/interlocutor/screens/InterlocutorDisplay/actions/index.js

```diff
@@ -0,0 +1,19 @@
+import moment from 'moment';
+
+import { GET_INTERLOCUTOR_ACTIVITY } from './types';
+import { executeSql } from 'yasav/src/Database';
+
+export function getInterlocutorActivity(interlocutorId) {
+  return (dispatch, getState) => {
+    executeSql('SELECT title, description, activity_date, id FROM activity WHERE interlocutor_id = ?', [interlocutorId])
```

Comment:
Should propably directly return the `executeSql` statement (like you did in `InterlocutorAdd/actions/index.js`, this is a good convention as is can be used later to chain asynchronous actions, see [here](https://github.com/gaearon/redux-thunk/blob/master/README.md#composition) in makeSandwichesForEverybody function for details.